mmengine/visualization/vis_backend.py

```diff
@@ -360,10 +360,12 @@ def __init__(self,
                  save_dir: str,
                  init_kwargs: Optional[dict] = None,
                  define_metric_cfg: Optional[dict] = None,
+                 log_code_name: Optional[str] = None,
```

Comment:
Adding a new paramter in middle position may cause bc issue. Suggest moving it to the last.

Reply:
Got it, thanks for your suggestion.